### PR TITLE
Feature 74: Added userid to pictogram model to indicate the creating user

### LINKED ISSUE #74: As a developer I would like the pictogram model to have a the Id of the creating user
**Is your feature request related to a problem? Please describe.**
When implementing aau-giraf/weekplanner#273 it should only be possible for a guardian to delete pictograms they have added themselves. 

**Describe the solution you'd like**
Add a field to the pictogram model to store the userId

- Edited by @dkalaxdk to use correct referencing.

### PR DESCRIPTION
**Pull request by AdrianPlesner**
Closes #74

### CODE REVIEW

**Comment by Determinatoro** [comment]
Please use the following checklist, @Riasalit. Feel free to check it off as you go along.

**Code Design**
- [ ] The code is in the right place? (Both in terms of folder structure and class structure)
- [ ] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [ ] This code could not, to the best of my knowledge, have reused existing code
- [ ] The code does not introduce functionality that is necessary for solving the problem

**Code Readability and Maintainability**
- [ ] Names are meaningful and self-documenting
- [ ] It is understandable, by reading the code, what it does
- [ ] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [ ] All new functionality is covered by tests
- [ ] It is understandable, by reading the tests, what they do
- [ ] The tests covers sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [ ] The features that the code claims to implement, are actually implemented
- [ ] The code does not look like it has bugs
- [ ] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [ ] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [ ] There is well written code in this. Remember to tell the developer!

**Comment by Determinatoro** [comment]
Please use the following checklist, @ggAlexS. Feel free to check it off as you go along.

**Code Design**
- [ ] The code is in the right place? (Both in terms of folder structure and class structure)
- [ ] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [ ] This code could not, to the best of my knowledge, have reused existing code
- [ ] The code does not introduce functionality that is necessary for solving the problem

**Code Readability and Maintainability**
- [ ] Names are meaningful and self-documenting
- [ ] It is understandable, by reading the code, what it does
- [ ] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [ ] All new functionality is covered by tests
- [ ] It is understandable, by reading the tests, what they do
- [ ] The tests covers sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [ ] The features that the code claims to implement, are actually implemented
- [ ] The code does not look like it has bugs
- [ ] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [ ] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [ ] There is well written code in this. Remember to tell the developer!

File: lib/models/pictogram_model.dart [hunk 1]
```diff
@@ -65,7 +71,11 @@ class PictogramModel implements Model {
...
     }
 
+    if(userId != null){
+      result['userId'] = userId;
+    }
+
     return result;
   }
-
+// hej Oliver
```
Comment (Riasalit) [inline lib/models/pictogram_model.dart:80]: ehhhh, tænker ikke det her skal med

**Review by Riasalit — CHANGES_REQUESTED** [review]

File: lib/models/pictogram_model.dart (hunk 1 above)
Comment (AdrianPlesner) [reply lib/models/pictogram_model.dart:80]: Det har du vidst ret i. :)

**Review by ggAlexS — APPROVED** [review]
seems fair

**Review by Riasalit — APPROVED** [review]

**Comment by AdrianPlesner** [comment]
Vi har flyttet den til develop så I skal lige approve igen. Beklager ulejligheden. :)

**Review by ggAlexS — CHANGES_REQUESTED** [review]
Can there be written af test including the userid?

**Review by ggAlexS — APPROVED** [review]

**Review by Riasalit — APPROVED** [review]

**Pull request merged**